### ISSUE TITLE
get_status causing RuntimeError: cannot reuse already awaited coroutine

### ISSUE THREAD

**Issue by bobloy** [Plutoberth/python-launch-library#9]
Run this code to replicate:
```py
import asyncio
import launchlibrary as ll

async def launch_test():
    api = ll.Api()
    launches = await api.async_next_launches(2)
    await launches[0].get_status()
    await launches[1].get_status()  # Runtime error here

loop = asyncio.get_event_loop()
loop.run_until_complete(launch_test())
```

Causes
```
Traceback (most recent call last):
  File "<input>", line 10, in <module>
  File "C:\Users\Bobloy\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "<input>", line 7, in launch_test
  File "C:\Users\Bobloy\Documents\GitHub\Fox-V3\venv\lib\site-packages\launchlibrary\async_models.py", line 86, in get_status
    launch_status = await AsyncLaunch._get_status_for_id(self.network, self.status)
RuntimeError: cannot reuse already awaited coroutine```

**Comment by Plutoberth** [comment]
Pretty sure that this is due to the way I cache, should be a pretty simple fix

**Comment by Plutoberth** [comment]
Fixed in version `1.0.6` :)
